### PR TITLE
changes related to .atn to actionJson API and Ray's video emabedded

### PR DESCRIPTION
**Pull request by khound**
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] I have signed the [Adobe Open Source CLA](https://opensource.adobe.com/cla.html).
- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

**Review by kmikawa — COMMENTED** [review]
Isn't "actionJSON" the official term?
This page also uses "actionJSON" instead of "ActionJson"
https://developer.adobe.com/photoshop/uxp/2022/ps_reference/media/batchplay/

**Comment by khound** [comment]
> Isn't "actionJSON" the official term? This page also uses "actionJSON" instead of "ActionJson" https://developer.adobe.com/photoshop/uxp/2022/ps_reference/media/batchplay/

Looks like. Let me change it to actionJSON

File: static/PITS/post_action_json_create.schema.json [hunk 1]
```diff
@@ -0,0 +1,40 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "$id": "post_action_json_create.schema.json",
+  "title": "post_action_json_create",
+  "description": "Initiates an asynchronous job to convert an .atn file to actionJson",
```
Comment (kmikawa) [inline static/PITS/post_action_json_create.schema.json]: actionJson -> actionJSON

File: src/pages/features/index.md [hunk 2]
```diff
@@ -39,26 +39,35 @@ Here are examples of submitting and executing Photoshop Actions.
 In this example we applied a custom Action we created called "Graphic Design."
 ![alt image](./psactions_example.png?raw=true "Original Image")
 
-### ActionJSON
+### ActionJson
```
Comment (kmikawa) [inline src/pages/features/index.md:42]: ActionJson -> actionJSON?

**Review by kmikawa — APPROVED** [review]

File: src/pages/features/index.md (hunk 2 above)
Comment (khound) [reply src/pages/features/index.md:42]: This I kept  `A` upper case as it is a heading?

File: static/PITS/post_action_json_create.schema.json (hunk 1 above)
Comment (khound) [reply static/PITS/post_action_json_create.schema.json]: I did change it. Thanks

**Pull request merged**